lib/src/cpp/common/LocatorList.cpp

```diff
@@ -162,12 +164,85 @@ void set_kind(
 
 void set_port(
         utils::ParseXML& manager,
```

Comment:
I think this dependency, `ParseXML` should be included following our criteria of including what we use. On the other hand, I suggest not including the Xerces dependency as it is going to be wrapped around `ParseXML` so it should not appear in this file once the infrastructure refactor is done.